go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/onsi/ginkgo v1.10.2
 	github.com/onsi/gomega v1.7.0
-	github.com/openebs/api v0.0.0-20200429072019-650209fba28b
+	github.com/openebs/api v1.10.0-RC1.0.20200602151240-2b7d2bdbe1ef
```

Comment:
If I am not wrong if we do `go get github.com/openebs/api@v1.10.0` then it will point to v1.10.0 branch which doesn't point to latest changes in API.

Reply:
It will point to v1.10.0 tag, not branch. If current code is using the changes from master then existing change is ok, but if doesn't then we should change it to latest tag.